mappings/net/minecraft/dispenser/impl/BehaviorDispenseItem.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS none/cr net/minecraft/dispenser/impl/BehaviorDispenseItem
```

Comment:
Why not DispenserBehaviorItem? The names, IMHO, should aid in seeing the hierarchy of inheritance.